tests/src/test/scala/cats/tests/NestedSuite.scala

```diff
@@ -153,4 +153,10 @@ class NestedSuite extends CatsSuite {
     checkAll("Nested[ListWrapper, Option, ?]", MonoidKTests[Nested[ListWrapper, Option, ?]].monoidK[Int])
     checkAll("MonoidK[Nested[ListWrapper, Option, ?]]", SerializableTests.serializable(MonoidK[Nested[ListWrapper, Option, ?]]))
   }
+
+  {
+    //Distributive composition
+    checkAll("Nested[Function0, Function0, ?]", DistributiveTests[Nested[Function0, Function0, ?]].distributive[Int, Int, Int, Option, Function0])
```

Comment:
I ran into an error about Eq instances because `distributive` in the Tests needs to be able to compare. I gave up and thought this was a reasonable compromise.

Reply:
Ah, that's due to the Eq instance for Function1 isn't provided by default
You can do the following 

```scala
    import cats.laws.discipline.eq._
    //Distributive composition
    checkAll("Nested[Function1[Int, ?], Function0, ?]", DistributiveTests[Nested[Function1[Int, ?], Function0, ?]].distributive[Int, Int, Int, Option, Function0])

```
That passes.